src/test/java/com/karankumar/bookproject/backend/statistics/util/StatisticTestUtils.java

```diff
@@ -66,31 +74,36 @@ private static void init(BookService bookService,
         bookService.deleteAll();
         savedBooks.clear();
         totalRating = 0.0;
+        thisYearRating = 0.0;
         StatisticTestUtils.predefinedShelfService = predefinedShelfService;
     }
 
     public static void addReadBook(BookService bookService,
                                    PredefinedShelfService predefinedShelfService) {
         init(bookService, predefinedShelfService);
-        createReadBook("Book", RatingScale.EIGHT, BookGenre.ANTHOLOGY, 200);
+        createReadBook("Book", RatingScale.EIGHT, BookGenre.ANTHOLOGY, 200,LocalDate.now());
     }
 
-    private static Book createReadBook(String bookTitle, RatingScale rating, BookGenre bookGenre, int pages) {
+    private static Book createReadBook(String bookTitle, RatingScale rating, BookGenre bookGenre, int pages,LocalDate startedReading) {
         PredefinedShelf readShelf = predefinedShelfService.findReadShelf();
-        Book book = createBook(bookTitle, readShelf, bookGenre, pages);
+        Book book = createBook(bookTitle, readShelf, bookGenre, pages,startedReading);
         book.setRating(rating);
 
         saveBook(book); // this should be called here & not in createBook()
         updateTotalRating(rating);
+        if (startedReading.getYear() == LocalDate.now().getYear()){
```

Comment:
This can use the method in date utils. Also, could you please ensure there is a space before the curly brace?